compiler/data/class-data.cpp

```diff
@@ -27,6 +28,7 @@ const char *ClassData::NAME_OF_TO_STRING = "__toString";
 
 ClassData::ClassData(ClassType type) :
   class_type(type),
+  kphp_json_tags(new kphp_json::ClassTags),
```

Comment:
It seems not logical that this field is non-null for all classes. It's better to allocate memory only when a class has @kphp-json tag.
See below, it's the same for fields

Reply:
fixed